src/realm/query_expression.hpp

```diff
@@ -4021,8 +4021,15 @@ class MinMaxAggregateOperator {
 public:
     void accumulate(T value)
     {
-        if (!is_nan(value) && (!m_result || Compare()(value, *m_result))) {
-            m_result = value;
+        if constexpr (std::is_same_v<T, Mixed>) {
+            if (!value.is_null() && (!m_result || Compare()(value, *m_result))) {
+                m_result = value;
+            }
+        }
+        else {
+            if (!is_nan(value) && (!m_result || Compare()(value, *m_result))) {
+                m_result = value;
+            }
         }
```

Comment:
```suggestion
        if constexpr (std::is_same_v<T, Mixed>) {
             if (value.is_null())
                 return
         }
         else {
             if (is_nan(value))
                 return;
         }
         
         if (!m_result || Compare()(value, *m_result))
             m_result = value;
```

Reply:
I will update master->v11 once this is merged.